applications/incompressible_navier_stokes/flow_past_cylinder/application.h

```diff
@@ -391,9 +396,9 @@ class Application : public ApplicationBase<dim, Number>
     pp_data.output_data.time_control_data.is_active        = this->output_parameters.write;
     pp_data.output_data.time_control_data.start_time       = start_time;
     pp_data.output_data.time_control_data.trigger_interval = (end_time - start_time) / 20.0;
-    pp_data.output_data.directory          = this->output_parameters.directory + "vtu/";
-    pp_data.output_data.filename           = name;
-    pp_data.output_data.write_divergence   = true;
+    pp_data.output_data.directory        = this->output_parameters.directory + "vtu/";
+    pp_data.output_data.filename         = name;
+    pp_data.output_data.write_divergence = this->param.grid.element_type == ElementType::Hypercube;
```

Comment:
Maybe create an issue as reminder?

Reply:
Will do. This is also related to the invermassoperator not working for simplices.

Reply:
is this already resolved or issued,  @peterrum @necioglu ?

Reply:
Inversemassoperator is now available for simplices, hence I changed the statement back to `...= true;`.